internal/generationtest/generation_test.go

```diff
@@ -48,9 +48,10 @@ func TestFullGeneration(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Could not convert gazelle binary path %s to absolute path. Error: %v", *gazelleBinaryPath, err)
 	}
+	testNames := map[string]bool{}
```

Comment:
Could you make this a `map[string]struct{}` instead? That's a more idiomatic version of a set.

Reply:
done.